setup.cfg

```diff
@@ -1,4 +1,4 @@
 [flake8]
-ignore = E501, E203, W503, W504
+ignore = E501, E203, W503, W504, E704
```

Comment:
flake8's E704 didn't like this kind of changes by black
![](https://github.com/user-attachments/assets/3c0979b6-c642-4812-a15b-1b293ab137ef)

Reply:
Yes, it should be disabled too: https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#e701-e704